pkg/core/blockchain.go

```diff
@@ -508,20 +524,6 @@ func (bc *Blockchain) jumpToStateInternal(p uint32, stage stateJumpStage) error
 		}
 		fallthrough
 	case stateJumpStarted:
-		// Replace old storage items by new ones, it should be done step-by step.
-		// Firstly, remove all old genesis-related items.
-		b := bc.dao.Store.Batch()
-		bc.dao.Store.Seek([]byte{byte(bc.dao.Version.StoragePrefix)}, func(k, _ []byte) {
-			// #1468, but don't need to copy here, because it is done by Store.
-			b.Delete(k)
-		})
-		b.Put(jumpStageKey, []byte{byte(oldStorageItemsRemoved)})
```

Comment:
`oldStorageItemsRemoved` enum should be removed.

Reply:
Linter is still complaining about it.